From f1f2306d1f17d0d182c115b3597db156a2894cee Mon Sep 17 00:00:00 2001
From: Marcello Sylvester Bauer <marcello.bauer@9elements.com>
Date: Thu, 12 Feb 2026 21:59:42 +0100
Subject: [PATCH] ARM: dts: hpe-gxp: add coretemp sensor node

Add device tree node for the GXP SoC internal temperature sensor.

Upstream-Status: Pending

Signed-off-by: Marcello Sylvester Bauer <marcello.bauer@9elements.com>
---
 arch/arm/boot/dts/hpe/hpe-gxp.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/hpe/hpe-gxp.dts b/arch/arm/boot/dts/hpe/hpe-gxp.dts
index b35542d465b4..ca0d446a084c 100644
--- a/arch/arm/boot/dts/hpe/hpe-gxp.dts
+++ b/arch/arm/boot/dts/hpe/hpe-gxp.dts
@@ -545,6 +545,11 @@ umac1: ethernet@c0005000 {
 			nvmem-cell-names = "mac-address";
 		};
 
+		coretemp: coretemp@c0000130 {
+			compatible = "hpe,gxp-coretemp";
+			reg = <0xc0000130 0x8>;
+		};
+
 		fan-keys {
 		compatible = "gpio-keys-polled";
 		poll-interval = <1000>;
